nntrainer/layers/permute_layer.cpp

```diff
@@ -57,7 +57,7 @@ void PermuteLayer::finalize(InitLayerContext &context) {
 
     /*** @todo deprecate this */
     direction_str = buildTrasposeString(direction);
-    rdirection_str = buildTrasposeString(direction);
+    rdirection_str = buildTrasposeString(reverse_direction);
```

Comment:
wonder why it did not have any problem.

Reply:
it doesn't have unittest, so never got caught I guess.